test/stubs/contentful/get-entries-campaign-with-overrides.json

```diff
@@ -0,0 +1,40 @@
+{
+  "sys": {
+    "type": "Array"
+  },
+  "total": 1,
+  "skip": 0,
+  "limit": 100,
+  "items": [
+    {
+      "sys": {
+        "space": {
+          "sys": {
+            "type": "Link",
+            "linkType": "Space",
+            "id": "owik07lyerdj"
+          }
+        },
+        "id": "6tlsrM2RTa40oMqwcIKSg6",
+        "type": "Entry",
+        "createdAt": "2017-02-15T20:30:31.426Z",
+        "updatedAt": "2017-05-12T15:20:14.634Z",
+        "revision": 10,
+        "contentType": {
+          "sys": {
+            "type": "Link",
+            "linkType": "ContentType",
+            "id": "campaign"
+          }
+        },
+        "locale": "en-US"
+      },
+      "fields": {
+        "campaignId": "7",
+        "gambitSignupMenuMessage": "Great! Little moments can have a big impact. Complimenting someone can turn their whole day around.\n\nDo this: post encouraging notes in places that can trigger low self-esteem, like school bathrooms.\n\nThen, text START to share a photo of the messages you posted (and you'll be entered to win a $1500 scholarship)! \n\nBonus: Text back a photo of your posted note by 11:59PM tonight and you can enter to win a gift card of your choice! (Winners announced this week). ",
```

Comment:
Ahh, yeah - editors aren't adding the {{cmd_reportback}} on overrides. Ideally they should be for if/when we ever change that start Command. On the default campaign, it's set to that (http://ds-mdata-responder-staging.herokuapp.com/v1/campaigns/819 is an example)